containers/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM quay.io/foreman/foreman-builder-base as builder
```

Comment:
is it builder or base? imho the naming are confusing.

Reply:
It's `builder-base`: the base for the `builder`. the `builder` itself is the container being built. I'm hesitant calling the `foreman-builder-base` as `builder` because, technically, the `builder` image is the product of `builder` phase in `Dockerfile` (same as base is not the fedora-minimal but fedora-miniman + additional changes we need)

Reply:
imho foreman-builder makes more sense :)

Reply:
Leaving some time for anyone else to express their opinion :)

Reply:
and to answer to more than oneliner, from a consumer POV:

foreman image:
temp builder container: FROM quay.io/foreman/foreman-builder
production container: FROM quay.io/foreman/foreman-base

just seems clear one is image is a base image, the other is the builder image, I understand the nuance that you build later on on top using another layered image, but think about the guy that browse the published images and try to figure out what is what.

Reply:
aha! I understand my cnofusion! there should be 3 published images -> 3 dockerfles! please have another look at https://github.com/ohadlevy/foreman/commit/2a07924576ea1c6af828240a5ea2f66aaab822ff

Reply:
I'm ok with 3 Dockerfiles